tools/manifest-templator/manifest-templator.go

```diff
@@ -219,6 +222,7 @@ func main() {
 		data.Verbosity = "{{.Verbosity}}"
 		data.CsvVersion = "{{.CsvVersion}}"
 		data.QuayRepository = "{{.QuayRepository}}"
+		data.RunbookURLTemplate = "{{.RunbookURLTemplate}}"
```

Comment:
@enp0s3 Is this line correct?

Reply:
@assafad Actually this is redundant because we don't expose `RUNBOOK_URL_TEMPLATE` to the virt-operator j2 template.

Reply:
Removed